content/300-guides/025-migrate/800-production-troubleshooting.mdx

```diff
@@ -135,11 +135,15 @@ In this case, you need to create a migration that takes your production database
 
   - Run the following `prisma migrate diff` command:
 
-    ```bash
-     npx prisma migrate diff --from-url "$DATABASE_URL_PROD" --to-migrations ./migrations --script > backward.sql
+    ```terminal wrap
+     npx prisma migrate diff \
+      --from-url "$DATABASE_URL_PROD" \
+      --to-migrations ./prisma/migrations \ 
+      --shadow-database-url $SHADOW_DATABASE_URL \
+      --script > backward.sql
```

Comment:
I don't this notation works on Windows.

Reply:
If using `cmd.exe` it won't work indeed but given that it's in many places like this it should be its own issue.
See https://superuser.com/a/1222011 the continuation character is different.

Reply:
TIL